README.md

```diff
@@ -187,6 +188,17 @@
 * [Simple Nixos Mailserver](https://gitlab.com/simple-nixos-mailserver/nixos-mailserver) - A complete mailserver, managed with NixOS modules.
 * [Stylix](https://github.com/danth/stylix) - System-wide colorscheming and typography for NixOS.
 
+## NixOS Configuration Editors
+
+### Desktop apps
+
+* [nixos-conf-editor](https://github.com/vlinkz/nixos-conf-editor) - Desktop app in Rust and GTK.
+* [nix-software-center](https://github.com/vlinkz/nix-software-center) - Install and manage Nix packages. Desktop app in Rust and GTK.
+
+### Webinterface
+
+* [mynixos.com](https://mynixos.com/) - Create and share software configurations using the NixOS ecosystem.
```

Comment:
I tried the tool and it looks like it allows you to:

- Create Nix flakes
- Manage flake inputs
- Create outputs of the following types:
  - Development shell
  - Home Manager config
  - Home Manager module
  - NixOS config
  - NixOS module
  - Package using [nixos-generate](https://github.com/nix-community/nixos-generators) to create an image from one of the configurations
- Search for packages from flakes in the flake registry and add them to one of the `devShell`s, configurations or modules
- Search for NixOS options from flakes in the flake registry and add them to one of the configurations or modules
- Commit the current state to create a new version in a rudimentary version control system.
- Download the flake as a tarball

It does not appear to be possible to modify the flake externally and then upload it back to the editor so it pretty much locks you to the on-line editor if you do not want to manually sync your changes.

So I would change the description to something like “Create and manage Nix flakes (including NixOS and home-manager configurations and modules) using an on-line editor. It is not possible to edit the data externally and upload them to the app.”

Reply:
yepp, its just a config generator, not a config editor

... because parsing arbitrary nix expressions is hard.
probably the internal storage format is not nix, but something static like json.
thats what i mean with "creating graphical nix editors is hard"